New package: thinkpad-scripts-4.11.0.

non-Jedi opened the pull request:
Since this is only useful on the specific Thinkpad models in question, should I set `only_for_archs="x86_64 x86_64-musl i686 i686-musl"`?

maxice8 commented on srcpkgs/thinkpad-scripts/template:
```diff
@@ -0,0 +1,19 @@
+post_install() {
+	python3 setup.py install --single-version-externally-managed --root="${DESTDIR}"
+	mv "${DESTDIR}/lib/udev" "${DESTDIR}/usr/lib/"
+	rmdir "${DESTDIR}/lib"
```
no need, this directory will be removed automatically.

maxice8 commented on srcpkgs/thinkpad-scripts/template:
```diff
@@ -0,0 +1,19 @@
+pkgname=thinkpad-scripts
+version=4.11.0
+revision=1
+build_style=gnu-makefile
```
shouldn't this be a python3-module then ?

non-Jedi replied:
Honestly, I didn't look closely enough to have any idea how the internals of this package work. I saw that it had a Makefile and just started with that and then later saw the installation instructions wanted you to do something with setup.py. I'll see if I can package it as `build_style=python3-module` tomorrow.

non-Jedi commented:
Pushed a commit addressing your comments @maxice8. Looks like I got your handle wrong in the commit message; sorry.

Let me know if it looks good, and you'd like me to squash and force push.

maxice8 commented:
Please squash them, all changes to a package should be in one commit
